public/resources/functions/inqueue-callback.protected.js

```diff
@@ -226,7 +228,8 @@ exports.handler = function (context, event, callback) {
           '/inqueue-callback?mode=mainProcess&callsid=' +
           callSid +
           '&cbphone=' +
-          explodePhone('format', phone),
+          explodePhone('format', phone) +
```

Comment:
Suggestion for a better query map builder:

```js
// Have this helper
const urlBuilder = (url, queries) => {
  const params = new URLSearchParams();
  Object.entries(queries).forEach(([key, value]) => params.append(key, value));
  return `${url}?${params}`;
}

// then your `action` here can be
const queries = {
  mode: 'mainProcess',
  callsid: callSid,
  cbphone: explodePhone('format', phone),
};
if (event.taskSid) {
  queries.taskSid = event.taskSid;
}
const action = urlBuilder(`${domain}/inqueue-callback`, queries);
```

Easier to read IMO

Reply:
Agree with you! Is that of with you if I incorporate these changes (as well as the one in #15  in another PR? I would like to keep this PR only for the TR API calls optimisation

Reply:
Raised #21